flask_oidc/__init__.py

```diff
@@ -762,165 +182,6 @@ def logout(self):
-        :param render_errors: Whether or not to eagerly render error objects
-            as JSON API responses. Set to False to pass the error object back
-            unmodified for later rendering.
-        :type render_errors: callback(obj) or None
-
-        .. versionadded:: 1.0
-        """
-
-        def wrapper(view_func):
-            @wraps(view_func)
-            def decorated(*args, **kwargs):
-                token = None
-                if 'Authorization' in request.headers and request.headers['Authorization'].startswith('Bearer '):
-                    token = request.headers['Authorization'].split(None,1)[1].strip()
-                if 'access_token' in request.form:
-                    token = request.form['access_token']
-                elif 'access_token' in request.args:
-                    token = request.args['access_token']
-
-                validity = self.validate_token(token, scopes_required)
-                if (validity is True) or (not require_token):
-                    return view_func(*args, **kwargs)
-                else:
-                    response_body = {'error': 'invalid_token',
-                                     'error_description': validity}
-                    if render_errors:
-                        response_body = json.dumps(response_body)
-                    return response_body, 401, {'WWW-Authenticate': 'Bearer'}
-
-            return decorated
-        return wrapper
-
-    def _get_token_info(self, token):
-        # We hardcode to use client_secret_post, because that's what the Google
-        # oauth2client library defaults to
-        request = {'token': token}
-        headers = {'Content-type': 'application/x-www-form-urlencoded'}
-
-        hint = current_app.config['OIDC_TOKEN_TYPE_HINT']
-        if hint != 'none':
-            request['token_type_hint'] = hint
-
-        auth_method = current_app.config['OIDC_INTROSPECTION_AUTH_METHOD'] 
-        if (auth_method == 'client_secret_basic'):
-            basic_auth_string = '%s:%s' % (self.client_secrets['client_id'], self.client_secrets['client_secret'])
-            basic_auth_bytes = bytearray(basic_auth_string, 'utf-8')
-            headers['Authorization'] = 'Basic %s' % b64encode(basic_auth_bytes).decode('utf-8')
-        elif (auth_method == 'bearer'):
-            headers['Authorization'] = 'Bearer %s' % token
-        elif (auth_method == 'client_secret_post'):
-            request['client_id'] = self.client_secrets['client_id']
-            if self.client_secrets['client_secret'] is not None:
-                request['client_secret'] = self.client_secrets['client_secret']
-
-        resp, content = httplib2.Http().request(
-            self.client_secrets['token_introspection_uri'], 'POST',
-            urlencode(request), headers=headers)
-        # TODO: Cache this reply
-        return _json_loads(content)
+        session.pop("token", None)
+        session.pop("userinfo", None)
+        return redirect("/")
```

Comment:
It does not look like the previous `logout()` method redirected the user anywhere, did it?